src/typescript/views/todoView.ts

```diff
@@ -29,6 +33,25 @@ class TodoView {
 
         this.todoListEl.innerHTML += todoTemplate;
     };
+
+    addEventSearch = (): void => {
+        this.btnSearchEl?.addEventListener('click', async (e: MouseEvent) => {
+            e.preventDefault();
+
+            const value: string = this.inputSearchEl.value;
+
+            const nameTodos: NodeList = document.querySelectorAll('.todo-info-name');
+
+            nameTodos.forEach((nameTodo) => {
+                (nameTodo.parentNode?.parentNode as HTMLElement).classList.remove('todo-hidden');
+                if (nameTodo.textContent?.toLocaleLowerCase().includes(value)) {
+                    (nameTodo.parentNode?.parentNode as HTMLElement).classList.add('todo-show');
```

Comment:
Create variable for element to add class, and for condition

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!